app/views/comments/_single_comment.html.erb

```diff
@@ -1,5 +1,5 @@
 <!-- START of single comment -->
-<li class="<% if single_comment.unreviewed? %>unreviewed <% end %>comment group <%= cycle :odd, :even %>" id="comment_<%= single_comment.id %>" role="article">
+<li class="<% if !single_comment.pseud.nil? && single_comment.pseud.user.official %>official <% end %><% if single_comment.unreviewed? %>unreviewed <% end %>comment group <%= cycle :odd, :even %>" id="comment_<%= single_comment.id %>" role="article">
```

Comment:
It might be possible to shorten or tidy this logic a bit, e.g. `if single_comment&.pseud&.user&.official` or `if single_comment.pseud.present? && single_comment.pseud.user.official`

Even better, though, would be defining a helper method. That way, we wouldn't need to duplicate so much across the various views or edit so many places if the logic for applying the `official` class changes.